doc/devel/coding_style_guideline.rst

```diff
@@ -52,56 +54,60 @@ example, interleaved with multi-line comments that contain explanations of the
-cross-referencing them, such as in [Garyfaillidis2014]_, and put them under the
-:ref:`references` section.
-* Cross-reference related examples and files. Use the ``.. _specific_filename:``
-convention to label a file at the top of it. Thus, other pages will be able to
-reference the file using the standard Sphinx syntax
-``:ref:`specific_filename```.
-* Use an all-caps scheme for acronyms, and capitalize the first letters of the
-long names, such as in *Constrained Spherical Deconvolution (CSD)*, except in
-those cases where the most common convention has been to use lowercase, such
-as in *superior longitudinal fasciculus (SLF)*.
-* As customary in Python, use lowercase and separate words with underscores for
-filenames, labels for references, etc.
-* When including figures, use the regular font for captions (i.e. do not use
-bold faces), unless otherwise required for a specific text part (e.g. a DIPY
-object, etc.).
-* When referring to relative paths, use the backquote inline markup convention,
-such as in ``doc/devel``. Do not add the greater-than/less-than signs to
-enclose the path.
+Particularly, in order to ease the contribution of examples and ``.rst``
+files, and with the consistency criterion in mind, beyond the numpy
+docstring standard aspects, contributors are encouraged to observe the
+following guidelines:
+
+  * The acronym for the Diffusion Imaging in Python toolkit should be written
+    as **DIPY**.
+  * The classes, objects, and any other construct referenced from the code
+    should be written with inverted commas, such as in *In DIPY, we use an
+    object called ``GradientTable`` which holds all the acquisition specific
+    parameters, e.g. b-values, b-vectors, timings and others.*
+  * Cite the relevant papers. Use the *[NameYear]* convention for
+    cross-referencing them, such as in [Garyfaillidis2014]_, and put them
+    under the :ref:`references` section.
+  * Cross-reference related examples and files. Use the
+    ``.. _specific_filename:`` convention to label a file at the top of it.
+    Thus, other pages will be able to reference the file using the standard
+    Sphinx syntax ``:ref:`specific_filename```.
+  * Use an all-caps scheme for acronyms, and capitalize the first letters of
+    the long names, such as in *Constrained Spherical Deconvolution (CSD)*,
+    except in those cases where the most common convention has been to use
+    lowercase, such as in *superior longitudinal fasciculus (SLF)*.
+  * As customary in Python, use lowercase and separate words with underscores
+    for filenames, labels for references, etc.
+  * When including figures, use the regular font for captions (i.e. do not use
+    bold faces), unless otherwise required for a specific text part (e.g. a
+    DIPY object, etc.).
+  * When referring to relative paths, use the backquote inline markup
+    convention, such as in ``doc/devel``. Do not add the
+    greater-than/less-than signs to enclose the path.
 
 
 """
```

Comment:
Do we really need this quotes?  I think you can remove them because they look strange as you can see on [this page](http://nipy.org/dipy/devel/coding_style_guideline.html)

Reply:
I think they make easier reading the text, standing out what is a  `code snippet`  or *any* other construct that belong to the structure of the toolkit. The reason why they are double inverted quotes instead of simple inverted quotes is due to the Sphinx syntax in the docstrings (if I remember well).

A question relative to the way dirs had to be described was raised in a [comment](https://github.com/nipy/dipy/pull/1314#issuecomment-321590604) #1314 .

Reply:
I think we are not talking about the same line. I am talking about the quotes [here](https://github.com/nipy/dipy/pull/1421/files#diff-e1343e5bdca90449ac1bf352283b04a4R113), line 113

Reply:
OK, now I got it. Sorry. My bad.